LICENSE.md

```diff
@@ -47,4 +47,5 @@ Machine_Maker <machine@machinemaker.me>
 Ivan Pekov <ivan@mrivanplays.com>
 Camotoy <20743703+Camotoy@users.noreply.github.com>
 Bjarne Koll <lynxplay101@gmail.com>
-```
+Callum Seabrook (BomBardyGamer) <callum.seabrook@prevarinite.com>
```

Comment:
This needs to match the email in the patch below

Reply:
God damn it. Why is my Git email set to `yutubebroz@gmail.com` still, I'm an idiot. Let me try and fix that (if you can even rebase and change the email)

Reply:
Sorted that, thanks